app/templates/components/public/social-links.hbs

```diff
@@ -1,5 +1,5 @@
-<div class="ui list mt-0 mb-0 pt-0 pb-0">
-  <div class="social-links mt-0 mb-0 pt-0 pb-0">
+<div class="ui list mt-0 mb-0 pt-0 pb-0 pl-0 pr-0">
+  <div class="social-links mt-0 mb-0 pt-0 pb-0 pl-0 pr-0">
```

Comment:
make a class like .pd-0 {
padding: 0 !important; 
}
instead of setting left, right. top , bottom setting 0 seperately

Reply:
Ok sir will do that, Thanks for recommending.